### PR TITLE
pngcheck: add page

### PR DESCRIPTION
**Pull request by rhapsodyn**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/pngcheck.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# pngcheck
+
+> Pngcheck verifies the integrity of PNG, JNG and MNG files (by checking the internal 32-bit CRCs, a.k.a. checksums, and decompressing the image data).
+> It can optionally dump almost all of the chunk-level information in the image in human-readable form.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Print summary (width & height & color depth):
```
Comment (bl-ue) [inline pages/common/pngcheck.md:7]:
```suggestion
- Print summary (width, height, and color depth):
```

File: pages/common/pngcheck.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`pngcheck foo.png`
```
Comment (bl-ue) [inline pages/common/pngcheck.md:9]:
```suggestion
`pngcheck {{image.png}}`
```

File: pages/common/pngcheck.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- Print colorized output:
+
+`pngcheck -c foo.png`
```
Comment (bl-ue) [inline pages/common/pngcheck.md:13]:
```suggestion
`pngcheck -c {{image.png}}`
```

File: pages/common/pngcheck.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+- Print most chuck data (IHDR & PLTE & IDAT & etc):
+
+`pngcheck -cvt foo.png`
```
Comment (bl-ue) [inline pages/common/pngcheck.md:17]:
```suggestion
`pngcheck -cvt {{image.png}}`
```

File: pages/common/pngcheck.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# pngcheck
+
+> Pngcheck verifies the integrity of PNG, JNG and MNG files (by checking the internal 32-bit CRCs, a.k.a. checksums, and decompressing the image data).
```
Comment (bl-ue) [inline pages/common/pngcheck.md:3]:
```suggestion
> Print detailed information about and verify PNG, JNG, and MNG files.
```

File: pages/common/pngcheck.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+> It can optionally dump almost all of the chunk-level information in the image in human-readable form.
```
Comment (bl-ue) [inline pages/common/pngcheck.md:4]:
```suggestion
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Hi @rhapsodyn, thank you for the new page.  I've left some comments below, for you to review.

**Review by bl-ue — APPROVED** [review]

File: pages/common/pngcheck.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# pngcheck
+
+> Print detailed information about and verify PNG, JNG, and MNG files.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Print summary (width, height, and color depth):
```
Comment (owenvoke) [inline pages/common/pngcheck.md:6]:
```suggestion
- Print a summary for an image (width, height, and color depth):
```

File: pages/common/pngcheck.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+
+`pngcheck {{image.png}}`
+
+- Print [c]olorized output:
+
+`pngcheck -c {{image.png}}`
+
+- Print detailed information:
```
Comment (owenvoke) [inline pages/common/pngcheck.md:14]:
```suggestion
- Print [v]erbose information for an image:
```

File: pages/common/pngcheck.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
+# pngcheck
+
+> Print detailed information about and verify PNG, JNG, and MNG files.
+> More information: <http://www.libpng.org/pub/png/apps/pngcheck.html>.
+
+- Print summary (width, height, and color depth):
+
+`pngcheck {{image.png}}`
+
+- Print [c]olorized output:
```
Comment (owenvoke) [inline pages/common/pngcheck.md:10]:
```suggestion
- Print information for an image with [c]olorized output:
```

**Review by owenvoke — APPROVED** [review]
LGTM! Just a couple of minor changes I'd suggest, but they aren't really necessary, just suggestions.

File: pages/common/pngcheck.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+`pngcheck -c {{image.png}}`
+
+- Print detailed information:
+
+`pngcheck -cvt {{image.png}}`
```
Comment (navarroaxel) [inline pages/common/pngcheck.md:16]: I think we could add some extra examples:
```suggestion
`pngcheck -cvt {{image.png}}`

- Receive an image from stdin and display detailed information:

`cat {{path/to/image.png}} | pngcheck -cvt`

- [s]earch for PNGs within a specific file and display information about them:

`pngcheck -s {{image.png}}`

- Search for PNGs within another file and e[x]tract them:

`pngcheck -x {{image.png}}`
```

**Review by navarroaxel — APPROVED** [review]

File: pages/common/pngcheck.md [hunk 11]
```diff
@@ -3,14 +3,26 @@
...
-- Print detailed information:
+- Print [v]erbose information for an image:
 
 `pngcheck -cvt {{image.png}}`
+
+- Receive an image from stdin and display detailed information:
+
+`cat {{path/to/image.png}} | pngcheck -cvt`
+
+- [s]earch for PNGs within a specific file and display information about them:
```
[the remark this answers is not in the document]
Comment (navarroaxel) [reply pages/common/pngcheck.md:22]: No. Looks like a PNG could be built by a PNG composition. Or you can hide PNG in another type of binary files

[the remark this answers is not in the document]
Comment (bl-ue) [reply pages/common/pngcheck.md:22]: Hmm...so a 'PNG' is actually just an object.  🤔 

I guess we should keep it as is then!

**Review by bl-ue — APPROVED** [review]

**Pull request merged**